src/components/linkcarousel/LinkCarousel.tsx

```diff
@@ -58,7 +58,11 @@ const LinkCarousel: React.FC = () => {
           />
         </CarouselSlide>
         <CarouselSlide>
-          <LinkCard text="Team Leaderboard" path="/" background="img4" />
+          <LinkCard
+            text="Team Leaderboard"
+            path="/team-leaderboard"
+            background="img4"
```

Comment:
what is img4? Where is it stored?

Reply:
In the linkcard, folder. I didn't make this component, I just added the path variable and prettier changed the indenting